src/components/EditorHeader/EditorHeader.tsx

```diff
@@ -157,9 +157,10 @@ export const EditorHeader: React.FC<{
   ])
 
   const onTransitionClick = useCallback(
-    (event) => {
+    (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
why did, apparently, implicit typing worked before? Or it didnt?

Reply:
Maybe we can specify here an appropriate union type to avoid @ts-ignore down below?

Reply:
The issue here is that the event is used on Button, and for react button doesn't have value attribute.

The best way to avoid the //@ts-ignore is to use data-value instead of the value attr.

Reply:
> why did, apparently, implicit typing worked before? Or it didnt?

its a new update by React that each event should be typed.

Reply:
but since the editor-header component got remove in the following pr https://github.com/Atypon-OpenSource/manuscripts-style-guide/commit/765ca9995371e7c83ca9bab2157c9f90cb3f69cd, no need to perform any thing here. Resolving!